apps/journeys-admin/src/components/Editor/Toolbar/Menu/Menu.spec.tsx

```diff
@@ -172,7 +124,6 @@ describe('Toolbar Menu', () => {
       getByRole('menuitem', { name: 'Create Template' })
     ).toBeInTheDocument()
     expect(getByRole('menuitem', { name: 'Language' })).toBeInTheDocument()
-    expect(getByRole('menuitem', { name: 'Analytics' })).toBeInTheDocument()
     expect(getByRole('menuitem', { name: 'Copy Link' })).toBeInTheDocument()
   })
 
```

Comment:
I don't see any logic to that requires a publisher and a template for this to show up, just template. This also means you only ned the `Template Settings` assertion
```suggestion
 it('should render template menu', async () => {
```